tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -10956,7 +10957,7 @@ bool SpirvEmitter::processHSEntryPointOutputAndPCF(
     if (hlsl::IsHLSLInputPatchType(param->getType())) {
       pcfParams.push_back(hullMainInputPatch);
     } else if (hlsl::IsHLSLOutputPatchType(param->getType())) {
-      pcfParams.push_back(hullMainOutputPatch);
+      continue;
```

Comment:
Similarly, add a comment here about why this param is being skipped. e.g.
Please add a comment about why this parameter should be skipped. e.g.
```cpp
// Since the output patch used in hull shaders is translated to a variable with Workgroup storage
// class, there is no need to pass the variable as function parameter in SPIR-V.
```